packages/portfolio-contract/src/portfolio.contract.ts

```diff
@@ -11,27 +11,37 @@ import type { ZCF } from '@agoric/zoe';
 import type { ResolvedPublicTopic } from '@agoric/zoe/src/contractSupport/topics.js';
 import type { Zone } from '@agoric/zone';
 import type { CopyRecord } from '@endo/pass-style';
-import { M, mustMatch } from '@endo/patterns';
+import { M } from '@endo/patterns';
+import { TimerServiceShape } from '@agoric/time';
 import { preparePortfolioKit, type LocalAccount } from './portfolio.exo.ts';
 import * as flows from './portfolio.flows.ts';
-import {
-  makeProposalShapes,
-  makeOfferArgsShapes,
-  type OfferArgsShapes,
-} from './type-guards.ts';
+import { makeProposalShapes, type OfferArgsShapes } from './type-guards.ts';
+import { AxelarChains } from './constants.js';
 
+const { keys } = Object;
 const trace = makeTracer('PortC');
 
 const interfaceTODO = undefined;
 
 const privateArgsShape = {
   ...(OrchestrationPowersShape as CopyRecord),
   marshaller: M.remotable('marshaller'),
-  contract: M.splitRecord({
+  contractAddresses: M.splitRecord({
     aavePool: M.string(),
     compound: M.string(),
     factory: M.string(),
+    usdc: M.string(),
   }),
+  axelarChainsMap: M.recordOf(
+    M.or(...keys(AxelarChains)),
+    M.splitRecord({
+      caip: M.string(),
+      // Axelar chain Ids differ between mainnet and testnet environments.
+      // Reference: https://github.com/axelarnetwork/axelarjs-sdk/blob/f84c8a21ad9685091002e24cac7001ed1cdac774/src/chains/supported-chains-list.ts
```

Comment:
this sort of doc is golden. Let's put it in `/** ... */` so that it shows up on hover.

Probably move this shape (and a corresponding type) in type-guards